src/strategies/spidStrategy.ts

```diff
@@ -67,6 +74,7 @@ const spidStrategy = async (
   } = {
     idp: {
       ...idpsMetadataOption,
+      ...idpsSpidValidatorOption,
       xx_servizicie_test: {
         cert: [
```

Comment:
can we put this configuration data into some other file (instead of inlining it here) ?

Reply:
Yes we can, but when we'll remove this code to use `io-spid-commons` this change will be lost. Can we postpone this change?